indico/queries/datasets.py

```diff
@@ -80,6 +80,8 @@ def __init__(self, id: int):
 
     def process_response(self, response) -> Dataset:
         response = super().process_response(response)
+        if not "dataset" in response or type(response) != dict:
```

Comment:
is it better to use `type` or `isinstance`? also should the checks be reversed? first type, then `in`

Reply:
actually this is all wrong it should be response["dataset"] and yeah instance, I won't lie I've been coding in another language recently with this symantic ;p